src/js/app.js

```diff
@@ -92,6 +92,7 @@ async function getDataForEntireDay(listOfTimestamps, type) {
 })();
 
 document.getElementById('openbtn').addEventListener('click', toggleSidebar);
+document.getElementById('tb-close').addEventListener('click', hideTextbox);
 document.getElementById('map').addEventListener('click', hideTextbox);
```

Comment:
The X on the contact box doesn't work. That's because

```js
document.getElementById('tb-close')
```
only returns the first element it finds. We have two `tb-close` buttons though (About & Contact box).
Ideally, we bind the click listener to all `tb-close` buttons on the page:

```js
// targets all elements with the class 'tb-close'
for (let element of Array.from(document.querySelectorAll('.tb-close'))) {
  element.addEventListener('click', hideTextbox);
}
```